.rodare.json

```diff
@@ -120,6 +120,12 @@
       "name": "Gruber, Bernhard Manfred",
       "orcid": "0000-0001-7848-1690",
       "type": "Other"
+    },
+    {
+      "affiliation": "Max Planck Institute for Plasma Physics (IPP)",
+      "name": "Schild, Nils",
+      "orcid": "0009-0000-5048-4814",
```

Comment:
@DerNils-git can you double check please if this is your ORCiD and if I spelled your affiliation properly/in the official way?